udata/core/user/models.py

```diff
@@ -196,9 +196,12 @@ def json_ld(self):
             '@type': 'Person',
             '@context': 'http://schema.org',
             'name': self.fullname,
-            'description': mdstrip(self.about),
         }
 
+        description = mdstrip(self.about)
+        if description != '':
```

Comment:
If self.about is None, mdstrip returns an empty string. I realized that when re-writing a unit test.